plugin/xcodebuild.vim

```diff
@@ -80,6 +80,14 @@ function! s:sdk()
   endif
 endfunction
 
+function! s:runner()
+  if exists('g:xcodebuild_runner')
+    return g:xcodebuild_runner
+  else
+    return '!'
```

Comment:
That's a good idea. I can extract the default flags used for xcpretty the same way. I'm going to do that in a different PR.